.circleci/config.yml

```diff
@@ -426,7 +427,111 @@ jobs:
           comment: "Failed to Clean up prnet"
           pr_number: << pipeline.parameters.pr_number >>
 
-  build-and-push-docker-image:
+  build-helm-chart:
+    environment:
+      - IMAGE: taraxa-node
+      - HELMCHART: taraxa-node
+    machine:
+      image: ubuntu-2204:2022.04.2
```

Comment:
No need to use this image anymore. This was used for the node build I guess but now we can just use the helm image, right?

Reply:
Refactored. This job was removed.

2 workflows realasing helm charts:
- development
- production 
released operate on the same private orb.

```
  # run this workflow for branches specified below
  build-helm-chart:
    jobs:
      - helm-release/chart-publish:
          develop: true # means, that this is development release, no-verify and artifact -> ${HELMCHART_VERSION}-sha.${SHORT_GIT_HASH}
          charts-dir: charts
          chart-name: taraxa-node
          chartmuseum-url: https://charts.gcp.taraxa.io
          chartmuseum-username: HELM_REPO_USERNAME
          chartmuseum-password: HELM_REPO_PASSWORD
          context:
            - CHARTSREPO
          filters:
            branches:
              only: 
              - /^chart\/.*/
              - /^chore\/chart-.*/
              - /^fix\/chart-.*/
              - /^feature\/chart-.*/


  # run this workflow for tags, like chart-vX.Y.Z
  release-helm-chart:
    jobs:
      - helm-release/chart-publish:
          charts-dir: charts
          chart-name: taraxa-node
          chartmuseum-url: https://charts.gcp.taraxa.io
          chartmuseum-username: HELM_REPO_USERNAME
          chartmuseum-password: HELM_REPO_PASSWORD
          context:
            - CHARTSREPO
          filters:
            branches:
              ignore: /.*/
            tags:
              only: /^chart-v\d+.\d+.\d+/
```